packages/react-native/React/CoreModules/RCTAlertController.mm

```diff
@@ -35,9 +35,8 @@ - (void)show:(BOOL)animated completion:(void (^)(void))completion
 {
   UIUserInterfaceStyle style = self.overrideUserInterfaceStyle;
   if (style == UIUserInterfaceStyleUnspecified) {
-    style = RCTSharedApplication().delegate.window.overrideUserInterfaceStyle
-        ? RCTSharedApplication().delegate.window.overrideUserInterfaceStyle
-        : UIUserInterfaceStyleUnspecified;
+      UIUserInterfaceStyle overridenStyle = RCTSharedApplication().delegate.window.overrideUserInterfaceStyle;
```

Comment:
nit: typo
```suggestion
      UIUserInterfaceStyle overriddenStyle = RCTSharedApplication().delegate.window.overrideUserInterfaceStyle;
```

Reply:
why not using `RCTKeyWindow()` here as well? It is the same `RCTSharedApplication().delegate.window` as in `RCTAppearance`.

Reply:
True, we can use it here also. Fixed 👍